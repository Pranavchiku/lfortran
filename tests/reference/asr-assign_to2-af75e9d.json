{
    "basename": "asr-assign_to2-af75e9d",
    "cmd": "lfortran --indent --show-asr --implicit-typing --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/assign_to2.f90",
    "infile_hash": "a929e60f25d1d3c3b1ef485c7d9b5dca6f7bebcfffc66ed010f309d3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-assign_to2-af75e9d.stdout",
    "stdout_hash": "181ced04638b9edd70f71257aa9d28fe002827ff553caddf0b59566c",
    "stderr": "asr-assign_to2-af75e9d.stderr",
    "stderr_hash": "cc14fd2c6b80fef15e0d85c22927c334fbcc9d088adebabd2b549935",
    "returncode": 0
}