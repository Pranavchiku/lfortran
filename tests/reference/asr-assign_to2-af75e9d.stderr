warning: Implied save feature
 --> tests/../integration_tests/assign_to2.f90:2:1
  |
2 | integer :: i, sum, ZERO
  | ^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/assign_to2.f90:2:1
  |
2 | integer :: i, sum, ZERO
  | ^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/assign_to2.f90:2:1
  |
2 | integer :: i, sum, ZERO
  | ^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
